src/OrbitGl/CallstackThreadBar.cpp

```diff
@@ -260,13 +260,23 @@ std::string CallstackThreadBar::FormatCallstackForTooltip(const CallstackInfo& c
   const int top_n = std::min(max_lines, size) - bottom_n;
 
   for (int i = 0; i < top_n; ++i) {
-    result.append("<br/>" + SafeGetFormattedFunctionName(callstack, i, max_line_length));
+    if (callstack.IsUnwindingError() && i > 0) {
```

Comment:
I would add a comment like in `CallstackDataView::GetDisplayColor` saying explicitly that we know the first frame is always correct.

Reply:
Done.